packages/api/ecs/async-operation/tests/test-index.js

```diff
@@ -132,27 +133,15 @@ test('updateAsyncOperation updates databases as expected', async (t) => {
         id: t.context.asyncOperationId,
       }
     );
-  const dynamoResponse = await DynamoDb.get({
```

Comment:
Test setup still uses this `DynamoDb` import, seems like it can be deleted. Same for the `dynamoDbDocClient`. In total I'm still seeing 17 hits in this file for `dynamo`

Reply:
You're entirely right, I must have missed manually checking this file in my post-work pass.     Updated: e51155ff89 (resolving other commentary from this file related): e51155ff89